app/javascript/bundles/UI/components/Card/Card.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import css from './Card.scss';
+import classnames from 'classnames';
+
+const Card = ({ mainText, subText, completedText }) => {
+  return <div class='card card-main'>
```

Comment:
I think we should rename card-secondary and card-completed to card-body and card-footer respectively. This way the HTML is more readable.

Also, I think we should do the same with the title. Maybe we could add: `<div class='card-header'>` to encapsulate the header.

Maybe `mainText`, `subText` and `completedText` should be renamed to follow the same naming.